frontend/TvCv19.Frontend/ClientApp/projects/shared-caregiver/src/lib/nurses-doctors-menu/nurses-doctors-menu.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="main-menu-text"><p>Menu</p></div>
+
+<div class="menu">
+  <mat-action-list>
+    <div class="menu__item" mat-list-item routerLink="../patients">
+      <span class="menu__header">Patients</span>
+      <span>List of Patients</span>
+    </div>
+    <div class="menu__item" mat-list-item routerLink="./">
+      <span class="menu__header">Notifications</span>
+      <span>Escalated Cases</span>
+    </div>
+    <div class="menu__item" mat-list-item routerLink="../change-shift">
+      <span class="menu__header">Shift Change</span>
+      <span>Hand off Patients</span>
+    </div>
+    <div class="menu__item" mat-list-item (click)="itemRemoveClick()">
+      <!-- routerLink="../../2/patients"> -->
+      <span class="menu__header">QR Code</span>
+      <span>Display the QR code</span>
+    </div>
+    <div *ngIf="showQR" class="bedside">
+      <div
+        class="bedside__patient-list__patient"
+        routerLink="../patient/{{ patient.id }}"
+        *ngFor="let patient of patients"
+      >
+        <div fxLayout="row" fxLayoutAlign="center">
+          <span class="patient-name">{{ patient.name }}</span>
+          <img src="/assets/img/message-bubble.png" />
+        </div>
+        <div>
+          <qrcode
+            [qrdata]="patient.id"
```

Comment:
this needs to be a physician id

Reply:
updated to physician.id

Reply:
I thought this should be patient.id as it is presenting the QR codes of the physician's patients. Please correct me if I'm wrong